src/custom-elements/Field/Field.js

```diff
@@ -1,60 +1,89 @@
+import { addStyles } from '../../services/Utils/Utils.js';
+
 class Field extends HTMLElement {
   constructor() {
     super();
-    this.bases = ['', '', ''];
-  }
 
-  removeEventListeners = () => {
-    document.removeEventListener('dice:bases', this.handleRunners);
-    document.removeEventListener('dice:switch', this.handleReset);
-    document.removeEventListener('game:new', this.handleReset);
-  }
+    let styles = `
```

Comment:
What's the story with setting some of these styles in JS? The reason I ask is because I see you have a Game.css file so I could see changing an element's CSS class in JS but I'm wondering why styles are being declared in a few different spots

Reply:
NOT AT ALL LATE RESPONSE BUT..

it's kind of mid-transition to getting it out of the CSS file and putting it in with the component itself. I could use a plain stylesheet for it rather than how I'm doing it (appending it via script insertion), but I kind of like having the css inside of the component itself